### DISCUSSION TITLE
"Do some taxes" not working today

### DISCUSSION THREAD

**Discussion by SurfSats** [davetapley/helium-tax#74] [Q&A] [answered]
Not sure if it's just me but none of my accounts or hotspot addresses work.  I get error message "Couldn't find address, use (e.g. a1b2c3d4e5f6..) and not name (e.g. three-funny-words)".  I'm not using the funny words =) and tried running it on different browsers.

**Comment by davetapley** [comment, marked as answer]
@SurfPorn there is an ongoing incident, please check again when it is resolved:
https://status.helium.com/incidents/r3thdc720nzc
